pages/productos/index.tsx

```diff
@@ -1,54 +1,49 @@
 import axios from 'axios'
 import type { NextPage } from 'next'
-import { useEffect, useState } from 'react'
+import { useCallback, useState } from 'react'
 
-import EmptyList from '@/components/EmptyList'
-import ListHeader from '@/components/ListHeader'
-import Loading from '@/components/Loading'
-import useAuth from '@/hooks/useAuth'
+import ProductsContainers from '@/containers/Products/ProductsContainers'
 import { Product } from '@/services/GraphQL/products/types'
-import ProductList from '@/views/Products/List'
 import ClientOnly from '@/views/Shared/ClientOnly'
 
-const showProductList = (products: Product[]) =>
-  products?.length > 0 ? (
-    <ProductList products={products} />
-  ) : (
-    <EmptyList className="center-text" text="No hay productos" />
-  )
+interface ProductResponse {
+  products: Product[]
+  products_aggregate: {
+    aggregate: {
+      totalCount: number
+    }
+  }
+}
 
 const Products: NextPage = () => {
   const [products, setProducts] = useState<Product[]>([])
+  const [totalProducts, setTotalProducts] = useState(0)
```

Comment:
I don't think this is really needed, use products.length instead

Reply:
this is to save the total number of products stored in the DB, in the response there are only 15 products per page.

Reply:
AH got it, you are right